test/utils/sliced_template_test.py

```diff
@@ -0,0 +1,76 @@
+"""Tests for ccx_messaging.utils.sliced_template module."""
```

Comment:
can you add some unit tests using the timestamp too? that looks like the most complex part

Reply:
Ok, but I'll include them as part of the engine tests instead, as the slice template can work with anything, it's not related at all with the complex regular expression including the timestamp